vnext/Microsoft.ReactNative/Views/ActivityIndicatorViewManager.cpp

```diff
@@ -23,6 +23,7 @@ void ActivityIndicatorViewManager::GetNativeProps(const winrt::Microsoft::ReactN
 
   winrt::Microsoft::ReactNative::WriteProperty(writer, L"animating", L"boolean");
   winrt::Microsoft::ReactNative::WriteProperty(writer, L"color", L"Color");
+  winrt::Microsoft::ReactNative::WriteProperty(writer, L"size", L"string");
```

Comment:
This is for Paper, was this feature just not implemented before?

Reply:
I'm not sure, this was implemented in Paper but I'm guessing the JavaScript may have handled it before but with us using composition and using exact pixels, it seemed to override what was being done before.